public/index.html

```diff
@@ -81,6 +81,7 @@
     <link rel="stylesheet" href="fonts.css" type="text/css" />
     <script>
       window.EXCALIDRAW_ASSET_PATH = "/";
+      window.EXCALIDRAW_EXPORT_SOURCE = window.location.origin;
```

Comment:
is this needed since the source defaults to origin ?

Reply:
Nope we can remove this line.